backend/battle/tests.py

```diff
@@ -1,3 +1,46 @@
-# from django.test import TestCase
+from django.test import TestCase, Client
+from model_bakery import baker
+from users.models import User
+from battle.models import Battle
 
-# Create your tests here.
+'''
+class ListBattlesTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.battle = baker.make('battle.Battle')
+        self.user = User.objects.create(email='deborah.mendonca@vinta.com.br', password='admin')
+        self.user.set_password('admin')
```

Comment:
I don't think you need to set the password here after the previous line.

Reply:
I tested, it doesn't work when I set the password in the previous line, but I get success when I call the set_password(). So, I'm going to delete the unnecessary `password='admin' `